src/app/fdctl/config/default.toml

```
@@ -698,10 +698,11 @@ dynamic_port_range = "8900-9000"
     # parallel users.
     bank_tile_count = 2
 
-    # How many shred tiles to run.  Should be set to 2.  This is
+    # How many shred tiles to run.  Should be set to 1.  This is
     # configurable and designed to scale out for future network
-    # conditions but there is no need to run more than 2 shred tiles
-    # given current `mainnet-beta` conditions.
+    # conditions but there is no need to run more than 1 shred tile
+    # given current `mainnet-beta` conditions. There is however
```

```suggestion
    # given current `mainnet-beta` conditions.  There is however
```

nit: double spacing in this file